_includes/api/en/5x/req-secure.md

```diff
@@ -1,5 +1,3 @@
- A Boolean property that is true if a TLS connection is established. Equivalent to: 
+<h3 id='req.secure'>req.secure</h3>
  
```

Comment:
You can disable the particular eslint rule that's complaining with `<!-- eslint-disable no-unused-vars -->` before the offending line. That way the code will still be linted, but we can ignore that particular rule for this code block. 

With an inline code example, the linter will never check it.